src/qibo/models/circuit.py

```diff
@@ -1269,6 +1270,15 @@ def _update_draw_matrix(self, matrix, idx, gate, gate_symbol=None):
 
         return matrix, idx
 
+    def draw_mpl(self, scale=0.6, cluster_gates=True, save_file=None):
+
+        ax = MPLDrawer().plot_qibo_circuit(self, scale, cluster_gates)
+
+        if save_file:
+            MPLDrawer.save_fig(ax.figure, save_file)
+
+        return ax
```

Comment:
Do not make this function a method of `Circuit`, to avoid making `Circuit` depending on the `mpldrawer` module.

Moreover, this function is just calling another function, and optionally (with a flag) a second one.
This is definitely not modular, and it leads to concatenate the arguments of the two functions.

Instead, just encourage direct usage of the functions above. Even by choosing simpler names.
E.g.:
```py
fig, ax = qibo.ui.plot(circuit, scale=0.6)
fig.savefig("mycircuit.png")
```
is perfectly acceptable as user's code.